src/client/rooms/ClassroomPageContainer.tsx

```diff
@@ -33,6 +34,9 @@ const socket = socketIOClient("/");
 export const ClassroomPageContainer: React.FunctionComponent<Props> = (
     props: Props
 ) => {
+    const notification = new Audio("/public/notification.wav");
```

Comment:
Also should be in a ref, your making a new instance each update of the component.